code/obj/access_spawn.dm

```diff
@@ -22,6 +22,9 @@
 
 	proc/setup()
 		for (var/obj/machinery/M in src.loc)
+			if (istype(M,/obj/machinery/door/airlock) && M:modBySpawner == 0) //access spawn mapper QoL upgrade: no more accidental extra access
+				M.req_access = null
+				M:modBySpawner = 1
```

Comment:
```suggestion
			if (istype(M, /obj/machinery/door/airlock)) //access spawn mapper QoL upgrade: no more accidental extra access
				var/obj/machinery/door/airlock/A = M
				if (!A.modded_by_spawner)
					A.req_access = null
					A.modded_by_spawner = 1
```
changes it up to not use the : as its generally bad practice. instead you should typecheck, set a new var thats of the type you want, and then use that one.
also changes the var name to be more in line with our existing conventions